### PR TITLE
Admin text upload done

### PR DESCRIPTION
**Pull request by emilioaurea**
Kseniia and I did it ya'll!!!!

### CODE REVIEW

**Comment by emilioaurea** [comment]
Please don't fail. Please don't fail.

File: config/routes.rb [hunk 1]
```diff
@@ -28,4 +29,15 @@
   
   # admins routes
   get 'admins' => 'admins#index'
+  
+  #admin's text upload  
+  get 'admins/resources/index' => 'resources#index'
```
Comment (gavinksong) [inline config/routes.rb:34]: use resourceful routes instead.

**Review by gavinksong — CHANGES_REQUESTED** [review]
Delete commented code. Use resourceful routes. Clean up commits with `git rebase -i`: get rid of the weird test files (e.g. HW8). In the future, make break up your commits into smaller chunks (e.g. one commit for adding a gem, one commit for generating files, etc.).

**Comment by gavinksong** [comment]
It also makes more sense not to have the creation of a bug and the fix as separate commits in the same pull request.

File: config/routes.rb (hunk 1 above)
Comment (emilioaurea) [reply config/routes.rb:34]: Will do

Comment (emilioaurea) [reply config/routes.rb:34]: You mean resources resources right?

**Comment by emilioaurea** [comment]
- will delete commented code
- will use resourceful routes
- don't know what weird test files you are talking about
- don't know how to clean up commits and it's not necessary for the correctness or software design
- i object and demand a second opinion.

**Comment by emilioaurea** [comment]
will take commit feedback into consideration for next iteration

**Comment by emilioaurea** [comment]
Actually I see why Kseniia didn't do resources routes, it's because we would have to do resources for admins as well and we dont want to have unecessary routes.

**Comment by emilioaurea** [comment]
was there something else you ment by resourceful routes

**Comment by gavinksong** [comment]
I'm not sure what you mean when you say you would "have to do resources for admins as well."
I mean, do it like this:

```
resources :admins do
  collection do
    resources :resources
  end
end
```

**Comment by gavinksong** [comment]
[These](https://github.com/gavinksong/cs169-cucec/pull/57/commits/1b1166e9298cc604fe64d63a4f9c221e12888579#diff-c584ee842abb8d694d946c3ec74d79f3) are the junk files I am referring to. You can clean up commits with `git rebase -i`. I would argue that it is a necessary practice to have clean commits because if anyone cherry picks or reverts any individual change, they shouldn't end up with a load of crap.

**Comment by emilioaurea** [comment]
```
resources :admins do
  collection do
    resources :resources
  end
end
```

Ok regarding resourceful routes, if you do what you said would create CRUD action for admins which is not something we need. Thus, I believe it is the wrong design choice. @gavinksong This is backed by what @samuelkarani has mentioned in the past.

Secondly, regarding junk files, you made the same mistake I did and are looking at a commit that is not the most recent. Those junk files are from Kseniia's branch when I first started. I deleted those files in a further commit (8a0172277ce2772813134666c81addc66a5a48e3) and if you look at the most recent commit or the head of the branch those files are not present. 

Lastly, I agree that commits should be clean. However, nowhere in Git Practices does is this addressed. Moreover, I don't know what you want me to do. In addition, the only one who cherry picks is you. And thus, there are 5 other members of this team who won't be cherry picking and would like this code to run for this iteration. @KseniiaRyuma @yadelab @baoskee  If this is not the case, I would gladly edit the commits, just let me know how. Nonetheless, I want feedback from the whole team.

**Comment by emilioaurea** [comment]
Wait are you saying to clean up a commit I should delete those files from that commit? Why didn't you just say that?

**Comment by gavinksong** [comment]
Yeah, that is what I meant. That's what I've been saying.

**Comment by emilioaurea** [comment]
I still don't know how to do that and I have cs186 project just please accept my pull request so I can get on with my life. I'll tell you what. You accept mine and I'll accept yours. Heck, I'll even accept yours first and rebase off of master!

**Comment by gavinksong** [comment]
Also, admins is already a resourceful route. You have a route for `get 'admins' => 'admins#index'`. In fact, you can get rid of that too.

**Comment by gavinksong** [comment]
Just chill then. I'll do it.

**Comment by emilioaurea** [comment]
@gavinksong I rebased off of master bruh. check to see if it's all good.

**Comment by baoskee** [comment]
You smell

**Comment by emilioaurea** [comment]
Gavin's way or the highway.

**Pull request closed without merging**